front/app/components/Form/Components/Controls/MultiSelectCheckboxControl.tsx

```diff
@@ -9,15 +9,32 @@ import React, { useState } from 'react';
 
 // utils
 import { getLabel, sanitizeForClassname } from 'utils/JSONFormUtils';
+import { getOptions } from './controlUtils';
 
 // components
 import VerificationIcon from '../VerificationIcon';
-import { Box, Checkbox, Text } from '@citizenlab/cl2-component-library';
+import { Box, Checkbox, colors, Text } from '@citizenlab/cl2-component-library';
 import { FormLabel } from 'components/UI/FormComponents';
 import ErrorDisplay from '../ErrorDisplay';
+
+// i18n
 import { FormattedMessage } from 'utils/cl-intl';
 import messages from './messages';
-import { getOptions } from './controlUtils';
+
+// style
+import { darken } from 'polished';
+import styled from 'styled-components';
+
+const StyledBox = styled(Box)<{ checkboxBackgroundColor: string }>`
+  background-color: ${({ checkboxBackgroundColor }) =>
+    checkboxBackgroundColor ? checkboxBackgroundColor : colors.grey200};
+  &:hover {
+    background-color: ${({ checkboxBackgroundColor }) =>
+      checkboxBackgroundColor
+        ? darken(0.05, checkboxBackgroundColor)
+        : darken(0.05, colors.grey200)};
```

Comment:
Maybe a conversation we should have more broadly, but these `darken` methods are a bit confusing sometimes. If it's possible I think we should prefer using colors in the design system instead of generating new ones like this

Reply:
Hmm yes I can see your point. I think it might be difficult to do that for tenant colors, since they configure that in the platform :thinking: For hovering over elements with a tenant color we need a way to darken it slightly, and selecting from colors in our library might not offer a close enough match?

Maybe something to add to the next FE meeting agenda? :)

Reply:
Sounds good, I'll add it to the agenda :+1: